Add Kaptain to DKP 2.2 release notes and enable Kaptain Docs (#4375)
mesosphere/dcos-docs-site: pages/dkp/konvoy/2.2/release-notes/index.md

The unified DKP user interface provides a smooth experience independent of where you start your journey. Both DKP Essential and DKP Enterprise customers will have similar experiences in the User Interface, with DKP Enterprise customers gaining access to additional features and benefits simply by entering their DKP Enterprise [license key](/dkp/kommander/2.2/licensing/).

### Kaptain AI/ML, D2iQ’s AI/ML offering

> For better integration with DKP 2.2 you can launch Kaptain as a catalog application, and also support other platforms such as Amazon AWS EKS and Microsoft Azure AKS. This extends D2iQ’s openness to support Kubernetes platforms beyond DKP with Kaptain. Kaptain enables an organization to develop, deploy, and run entire ML workloads in production, at scale, with consistency and reliability.

Author: ```suggestion
For better integration with DKP 2.2, you can launch Kaptain as a catalog application. It also supports other platforms such as Amazon AWS EKS and Microsoft Azure AKS. Kaptain extends D2iQ’s ability to support Kubernetes platforms beyond DKP. It further enables an organization to develop, deploy and run entire ML workloads in production, at scale, with consistency and reliability.
```
